bundles/org.openhab.ui.homebuilder/web/src/restItems.js

```diff
@@ -65,7 +62,7 @@ export function addObjects(room, model, floorPrefix, roomObjects) {
             floorPrefix + room.value,
             GROUP_PREFIX + object.value
         ],
-        tags: addTags(object, model),
+        tags: object.tags,
```

Comment:
What if you add an object or a room by yourself?
You can type your own room name and it'll appear on the list.

Reply:
It goes fine now (also no errors on the console) and it should be even finer with the new method above, do you agree?

Reply:
No its not fine, let me do some more tests

Reply:
No it was already broken, it seems that the `onChanged` is not called when `onNewTag` is called, see:

http://demo.openhab.org:8080/homebuilder/index.html

Reply:
I will fix that

Reply:
FTR: https://github.com/vue-generators/vue-form-generator/issues/536